app/src/main/AndroidManifest.xml

```diff
@@ -10,20 +10,18 @@
         android:allowBackup="true"
         android:dataExtractionRules="@xml/data_extraction_rules"
         android:fullBackupContent="@xml/backup_rules"
-        android:icon="@mipmap/ic_launcher"
+        android:icon="@mipmap/ic_broccoli"
         android:label="@string/app_name"
-        android:roundIcon="@mipmap/ic_launcher_round"
+        android:roundIcon="@mipmap/ic_broccoli_round"
         android:supportsRtl="true"
-        android:theme="@style/Theme.Oipie"
-        tools:targetApi="31">
+        android:theme="@style/Theme.Oipie.Starting"
+        tools:targetApi="33">
         <activity
             android:name=".MainActivity"
             android:exported="true"
-            android:label="@string/app_name"
-            android:theme="@style/Theme.Oipie">
```

Comment:
should be  `android:theme="@style/Theme.Oipie.Starting"` also ¿?

Reply:
I thing is not necessary because we need to set the screen splash theme at <application> level, and reference from there to "Theme.Oipie"